CMakeLists.txt

```diff
@@ -162,7 +157,6 @@ else()
     endif()
 endif()
 
-include_directories ("${PROJECT_SOURCE_DIR}/include")
```

Comment:
Is this not required?

Reply:
I'm perplexed on how the test builds were successful across multiple platforms.  I've re-added this line.  Thanks for the catch.

Reply:
As am I...